exporter/elasticsearchexporter/integrationtest/exporter_test.go

```diff
@@ -15,41 +16,55 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
 )
 
-func TestLogExporter(t *testing.T) {
-	for _, tc := range []struct {
-		name string
-		// restartCollector restarts the OTEL collector. Restarting
-		// the collector allows durability testing of the ES exporter
-		// based on the OTEL config used for testing.
-		restartCollector bool
-		mockESFailure    bool
-	}{
-		{name: "basic"},
-		{name: "es_intermittent_failure", mockESFailure: true},
-		/* Below tests should be enabled after https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30792 is fixed
-		{name: "collector_restarts", restartCollector: true},
-		{name: "collector_restart_with_es_intermittent_failure", mockESFailure: true, restartCollector: true},
-		*/
-	} {
-		t.Run(tc.name, func(t *testing.T) {
-			runner(t, tc.restartCollector, tc.mockESFailure)
-		})
+func TestExporter(t *testing.T) {
+	for _, eventType := range []string{"logs", "traces"} {
+		for _, tc := range []struct {
+			name      string
+			eventType string
+			// restartCollector restarts the OTEL collector. Restarting
+			// the collector allows durability testing of the ES exporter
+			// based on the OTEL config used for testing.
+			restartCollector bool
+			mockESFailure    bool
+		}{
+			{name: "basic"},
+			{name: "es_intermittent_failure", mockESFailure: true},
+			/* Below tests should be enabled after https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30792 is fixed
```

Comment:
Good point on the `TODO`, I would like to keep the tests since it keeps the `restartCollector` option and the relevant code in context but happy to hear if others have any objection to it.